tbump.toml

```diff
@@ -0,0 +1,53 @@
+# Uncomment this if your project is hosted on GitHub:
```

Comment:
This file stores `tbump` configuration. 
`pyproject.toml` is supposedly supported but it didn't work. I opened up an issue https://github.com/dmerejkowsky/tbump/issues/103.